package.json

```diff
@@ -33,6 +33,7 @@
     "help": "dt -h"
   },
   "dependencies": {
+    "dt": "0.0.1"
```

Comment:
This is the 'diff tracker'. It wasn't included in the project so I had to install it -- I assumed that others would have the same problem down the line and I'm a big fan of reducing the effort involved in getting a project running the first time. Although I'm fine to rollback this commit if there's a reason that this isn't checked in